README.md

```diff
@@ -135,7 +135,38 @@ git clone https://github.com/NVIDIA/cccl.git
 nvcc -Icccl/thrust -Icccl/libcudacxx/include -Icccl/cub main.cu -o main
 ```
 > **Note**
-> Use `-I` and not `-isystem` to avoid collisions with the CCCL headers implicitly included by `nvcc` from the CUDA Toolkit. All CCCL headers use `#pragma system_header` to ensure warnings will still be silenced as if using `-isystem`, see https://github.com/NVIDIA/cccl/issues/527 for more information. 
+> Use `-I` and not `-isystem` to avoid collisions with the CCCL headers implicitly included by `nvcc` from the CUDA Toolkit. All CCCL headers use `#pragma system_header` to ensure warnings will still be silenced as if using `-isystem`, see https://github.com/NVIDIA/cccl/issues/527 for more information.
+
+#### Conda
+
+CCCL also provides conda packages of each release via the `conda-forge` channel:
+
+```bash
+conda install -c conda-forge cccl
```

Comment:
This will possibly cause problems if someone doesn't have `conda-forge` in their channel list and execute a subsequent installation command using only `defaults` for example.

Maybe we could link off to something stating to make sure you have conda-forge in your channel list?

Reply:
I’m not familiar with any documentation about that. I have always seen conda installation commands list the channel like this. Is there a canonical reference you would suggest? Or should we just show the command like `conda config --add channels conda-forge` before `conda install cccl`?

Reply:
This is a link from conda-forge's docs: https://conda-forge.org/docs/user/introduction/#how-can-i-install-packages-from-conda-forge

It has exactly what you wrote 😄

Reply:
Nice! I made this update in 368a9df11a2185ce2465b9adcdd5aa8318948188.